libs/core/src/lib/wizard/wizard-step/wizard-step.component.ts

```diff
@@ -76,6 +76,9 @@ export class WizardStepComponent implements OnChanges, AfterViewInit, OnDestroy
     @Input()
     isSummary = false;
 
+    @Input()
+    displaySummary = false;
```

Comment:
I do not think this needs to be an input. It can be a hidden property of the wizard-step component, and the wizard can iterate through ContentChildren to set this property when `[displaySummaryStep]="true"` on the wizard component.